src/themes/OLH/templates/cms/page.html

```diff
@@ -6,12 +6,27 @@
 
 {% block body %}
     <section id="content">
-      <div class="row column">
-        <h1 id="cms-title">{{ page.display_name }}</h1>
-        <hr />
-        <div id="cms">
-          {{ page.content|safe }}
+        <div class="row">
+            <div class="{% if page.display_toc %}border-right medium-8{% else %}medium-12{% endif %} columns">
+                <h1 id="cms-title">{{ page.display_name }}</h1>
+                <hr/>
+                <div id="main_article">
+                    {{ page.content|safe }}
+                </div>
+            </div>
+            {% if page.display_toc %}
+                <aside class="medium-4 columns sticky-container">
+                    <div class="sticky" data-sticky data-margin-top="0"
+                         data-anchor="content" data-sticky-on="large">
+                        <div class="section">
+                            <h3>{% trans "Table of Contents" %}</h3>
+                            <ul id="toc">
+
+                            </ul>
+                        </div>
+                    </div>
+                </aside>
+            {% endif %}
```

Comment:
This doesn't work so well on mobile screens, because the TOC is put at the end, where it would not be expected:

![image](https://github.com/BirkbeckCTP/janeway/assets/47217308/2b5c2ef7-8ac6-4f12-8e18-39ab0d70f29a)